services/src/integ-test/java/uk/gov/moj/sdt/services/messaging/MessageWriterIntTest.java

```diff
@@ -1,154 +1,149 @@
+public class MessageWriterIntTest extends AbstractIntegrationTest {
+    /**
+     * Logger object.
+     */
+    private static final Logger LOGGER = LoggerFactory.getLogger(MessageWriterIntTest.class);
+
+    private MessageReceiver messageReceiver;
+
+    /**
+     * Test method to test the sending of message.
+     *
+     * @throws JMSException         exception
+     * @throws InterruptedException exception
+     */
+    @Test
+    public void testQueueMessage() throws JMSException, InterruptedException {
+        // Get message writer from Spring.
+        final MessageWriter messageWriter =
+                (MessageWriter) this.applicationContext
+                        .getBean("uk.gov.moj.sdt.services.messaging.api.IMessageWriter");
+
+        final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");
+
+        // Clear any old messages off the queue.
+        while (true) {
+            // Read any old messages.
+            final ServiceBusReceivedMessage message = messageReceiver.receiveMessage("Test1Queue");
+            if (message == null) {
+                break;
+            }
+        }
+
+        // Send the first message.
+        final ISdtMessage message1 = new SdtMessage();
+        final String strMessage1 =
+                "TestMessage1" + dateFormat.format(new java.util.Date(System.currentTimeMillis()));
+        message1.setSdtRequestReference(strMessage1);
+        messageWriter.queueMessage(message1, "TEST1", false);
+
+        // Send the second message.
+        final ISdtMessage message2 = new SdtMessage();
+        final String strMessage2 =
+                "TestMessage2" + dateFormat.format(new java.util.Date(System.currentTimeMillis()));
+        message2.setSdtRequestReference(strMessage2);
+        messageWriter.queueMessage(message2, "TEST1", false);
+
+        // Read the two messages and ensure they are read back in the same order.
+        ServiceBusReceivedMessage message = messageReceiver.receiveMessage("Test1Queue");
+        if (message == null) {
+            Assert.fail("Test failed because JMS receive timed out.");
+        }
+
+        ISdtMessage sdtMessage = message.getBody().toObject(new TypeReference<SdtMessage>() {});
+        LOGGER.debug("Message Receieved 1 - " + sdtMessage.toString());
+        Assert.assertTrue(sdtMessage.getSdtRequestReference().equals(strMessage1));
+
+        message = messageReceiver.receiveMessage("Test1Queue");
+        sdtMessage = message.getBody().toObject(new TypeReference<SdtMessage>() {});
+        LOGGER.debug("Message Receieved2 - " + sdtMessage.toString());
+        Assert.assertTrue(sdtMessage.getSdtRequestReference().equals(strMessage2));
+
+    }
+
+    /**
+     * Test method to test failure behaviour when ACTIVE MQ not running.
+     *
+     * @throws JMSException exception
+     */
+    @Test
+    public void testActiveMqDown() throws JMSException {
```

Comment:
Should these now reference Azure Service Bus rather than ActiveMQ?

Reply:
Method has been renamed but the JavaDoc comment still references "ACTIVE MQ".

Reply:
JavaDoc comment has been changed but has lost the "not running" text from the end of it.  Needs to be added otherwise the sentence is incomplete!

Reply:
Confirmed comment has been updated.